docs/content/_index.md

```diff
@@ -24,9 +24,9 @@ title: "Fluid Framework"
 
   {{< cta
     linkClasses="play"
-    linkURL="/playground/"
+    linkURL="/docs/start/examples/"
     iconClasses="play"
-    linkText="Play with Fluid"
+    linkText="Examples"
```

Comment:
I'm not sure what else to put here now...

Reply:
This seems reasonable for now.